bundles/org.openhab.binding.sony/src/main/java/org/openhab/binding/sony/internal/simpleip/SimpleIpProtocol.java

```diff
@@ -0,0 +1,1485 @@
+/**
+ * Copyright (c) 2010-2020 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.sony.internal.simpleip;
+
+import java.io.File;
+import java.io.IOException;
+import java.net.SocketTimeoutException;
+import java.nio.charset.Charset;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Map.Entry;
+import java.util.Objects;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import org.apache.commons.lang.StringUtils;
+import org.apache.commons.lang.Validate;
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.jdt.annotation.Nullable;
+import org.eclipse.smarthome.config.core.ConfigConstants;
+import org.eclipse.smarthome.core.library.types.OnOffType;
+import org.eclipse.smarthome.core.library.types.PercentType;
+import org.eclipse.smarthome.core.library.types.StringType;
+import org.eclipse.smarthome.core.thing.ThingStatus;
+import org.eclipse.smarthome.core.thing.ThingStatusDetail;
+import org.eclipse.smarthome.core.transform.TransformationException;
+import org.eclipse.smarthome.core.transform.TransformationService;
+import org.openhab.binding.sony.internal.SonyUtil;
+import org.openhab.binding.sony.internal.ThingCallback;
+import org.openhab.binding.sony.internal.net.NetUtil;
+import org.openhab.binding.sony.internal.net.SocketChannelSession;
+import org.openhab.binding.sony.internal.net.SocketSession;
+import org.openhab.binding.sony.internal.net.SocketSessionListener;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * This is the protocol handler for the Simple IP System. This handler will issue the protocol commands and will
+ * process the responses from the Simple IP system. The Simple IP system is a little flacky and doesn't seem to handle
+ * multiple commands in a single session. For this reason, we create a single {@link SocketSession} to listen for any
+ * notifications (whose lifetime matches that of this handler) and then create separate {@link SocketSession} for each
+ * request. Special care must be taken to differentiate between a Control request result and the Inquiry/Notification
+ * results to avoid misinterpreting the result (the control "success" message will have all zeroes - which has a form
+ * that matches some inquiry/notification results (like volume could be interpreted as 0!).
+ *
+ * Additional documentation: https://pro-bravia.sony.net/develop/integrate/ssip/overview/
+ *
+ * @author Tim Roberts - Initial contribution
+ *
+ *
```

Comment:
blank lines

Reply:
FYI - did a scan on the solution and fixed 22 files that followed the "blank after author tag"...